XR_APILAYER_NOVENDOR_toolkit/fsr.cpp

```diff
@@ -0,0 +1,216 @@
+// MIT License
+//
+// Copyright(c) 2021-2022 Jean-Luc Dupiot - Reality XP
```

Comment:
Please add a similar Copyright line to all the other files you've also modified.

Reply:
Also, please add yourself to the Contributions section of the `README.md` as the co-author for this project ;)